localized/games/eggquiz/eggquizliteracy_levels_ur.tsv

```diff
@@ -1239,14 +1239,14 @@ FishTest_6_2	1	6200001, 6200002, 6200003, 6200004, 6200005		sentence_sentence		
 			6200002	sentence_word		صحیح جواب چنیں			heart	heart	hiert^heart^hart
 			6200003	soundonly_word					play	play	fly^clay^play
 			6200004	sentence_word		This is my ____.			this is my brother	brother	brother^mother^dad
-			6200005	sentence_word		Fred is in his _____.			fred is in his bedroom	bedroom	bedroom^kitchen^house
+			6200005	sentence_word		Fareed is in his _____.			fred is in his bedroom	bedroom	bedroom^kitchen^house
 											
 FishTest_6_2	2	6200006, 6200007, 6200008, 6200009, 6200010									
 			6200006	sentence_word		صحیح جواب چنیں			news	news	mews^vews^news
 			6200007	sentence_word		صحیح جواب چنیں			paper	paper	paper^peper^papir
 			6200008	soundonly_word					sweet	sweet	sweet^green^brown
 			6200009	sentence_word		This is my ____.			this is my mother	mother	brother^mother^dad
-			6200010	sentence_word		Cindy is in the _____.			cindy is in the kitchen	kitchen	bedroom^kitchen^house
+			6200010	sentence_word		Chanda is in the _____.			cindy is in the kitchen	kitchen	bedroom^kitchen^house
```

Comment:
Please properly review the changes made. The word is not replaced at many places